README.md

```diff
@@ -73,22 +95,7 @@ processes:
     config:
       start_command: '/usr/local/rvm/bin/rvm jruby-9.3.2.0 do bundle exec honeybadger exec traject -c ./lib/traject/config/sirsi_config.rb -s solr_writer.max_skipped=-1 -s log.level=debug -s log.file=log/traject_marc_bodoni_prod_indexer.log'
 ```
-daemon processes run continuously. you can use `eye info` in an ssh session to view status information:
-```sh
-$ eye info
 
-traject
-  workers
-    earthworks-stage-indexer_0 .... up  (12:07, 0%, 73Mb, <1218016>)
-    folio_dev_indexer_0 ........... up  (14:34, 6%, 137Mb, <1386323>)
-    marc_bodoni_dev_indexer_0 ..... up  (12:07, 0%, 2731Mb, <1223573>)
-    marc_morison_dev_indexer_0 .... up  (12:07, 0%, 1253Mb, <1223963>)
-    sw_dev_indexer_0 .............. up  (12:07, 0%, 75Mb, <1224054>)
-    sw_dev_indexer_1 .............. up  (12:07, 0%, 73Mb, <1224303>)
-    sw_preview_stage_indexer_0 .... up  (12:07, 0%, 73Mb, <1224638>)
-    sw_preview_stage_indexer_1 .... up  (12:07, 0%, 73Mb, <1224861>)
-```
-you can stop and start daemons with e.g. `eye stop sw_dev_indexer`. note that it may take some time for all the processes to start and stop. for more information on eye, use `eye help` or see the [eye wiki](https://github.com/kostya/eye/wiki).
```

Comment:
could we replace this with the equivalent systemd command to check service status? I imagine it's `sudo systemctl status traject` or something like that, but would be good to document. also copying the `systemctl start`/`stop` commands here would be nice

Reply:
I don't think they are really equivalent.  `systemctl status traject.target` is only going to show:
```
sudo systemctl status traject.target
● traject.target
     Loaded: loaded (/lib/systemd/system/traject.target; enabled; vendor preset: enabled)
     Active: active since Mon 2023-05-08 14:02:26 PDT; 1 day 19h ago

May 08 14:02:26 sw-indexing-stage-a.stanford.edu systemd[1]: Reached target traject.target.
```


that isn't going to give any of the information that eye provided.

Reply:
It seems like `list-dependencies` will do this (sort of):
```
sudo systemctl list-dependencies traject.target
traject.target
● ├─traject-earthworks_stage_indexer.1.service
● ├─traject-folio_dev_indexer.1.service
● ├─traject-folio_dev_indexer.2.service
● ├─traject-folio_dev_indexer.3.service
● ├─traject-folio_dev_indexer.4.service
● ├─traject-folio_dev_indexer.5.service
● ├─traject-folio_dev_indexer.6.service
● ├─traject-folio_dev_indexer.7.service
● ├─traject-folio_dev_indexer.8.service
● ├─traject-marc_bodoni_dev_indexer.1.service
● ├─traject-marc_morison_dev_indexer.1.service
● ├─traject-sw_dev_indexer.1.service
● ├─traject-sw_dev_indexer.2.service
● ├─traject-sw_preview_stage_indexer.1.service
● └─traject-sw_preview_stage_indexer.2.service
```
at least you get the green dot for the ones that are running, which seems useful enough to document.

Reply:
I don't have sudo access to that. How do you?
```
sudo systemctl list-dependencies traject.target
[sudo] password for indexer:
```

Reply:
Ah, well, then the sudo isn't going to do anything. Anyways, I made a PR for that to puppet.  I didn't know about `list-dependencies` so thanks for brining that to my attention.